plugins/content_distribution/providers/tvinci/lib/api/KalturaTvinciDistributionJobProviderData.php

```diff
@@ -35,10 +35,11 @@ public function __construct(KalturaDistributionJobData $distributionJobData = nu
 
 		$feedHelper = new TvinciDistributionFeedHelper($distributionJobData->distributionProfile, $fieldValues);
 		$feedHelper->setEntryId( $entry->getId() );
+		$feedHelper->setReferenceId($entry->getReferenceID());
```

Comment:
if the MRSS solution will take place this code change will be redundant so I am leaving it as is for now